### PR TITLE
detect flex direction/justify-content/align-items based on all selected elements

### PR DESCRIPTION
**Pull request by bkrmendy**
## Problem
In the inspector-strategies backed inspector controls, `flex-direction`, `justify-content`, `align-items` and `display(: flex)` were determined only by inspecting the first selected element, disregarding the rest. This lead to misleading UI states.

## Fix
Check all selected elements for the props in question, and
- if they are set to the same value, display that value on the UI
- otherwise, display a default value on the UI

## After
null_state

## Before
image

### CODE REVIEW

**Review by balazsbajorics — APPROVED** [review]

File: editor/src/components/inspector/inspector-common.tsx [hunk 1]
```diff
@@ -61,23 +60,42 @@ function stringToFlexDirection(str: string | null): FlexDirection | null {
...
-export function detectFlexAlignJustifyContent(
+const DefaultJustifyContentFlexAlign: [FlexJustifyContent, FlexAlignment] = [
+  'flex-start',
+  'flex-start',
+]
+
+function detectFlexAlignJustifyContentOne(
   metadata: ElementInstanceMetadataMap,
   elementPath: ElementPath,
 ): [FlexJustifyContent, FlexAlignment] {
```
Comment (Rheeseyb) [inline editor/src/components/inspector/inspector-common.tsx:71]: I must have overlooked or missed this before, and I don't want to block this PR on something unrelated, but please can we replace this return value with an object

File: editor/src/components/inspector/inspector-common.tsx [hunk 2]
```diff
@@ -87,17 +105,48 @@ export function filterKeepFlexContainers(
...
+    ? allDetectedMeasurement[0]
+    : DefaultFlexDirection
+}
+
+export function detectAreElementsInFlexLayout(
+  metadata: ElementInstanceMetadataMap,
+  elementPaths: Array<ElementPath>,
+): boolean {
+  return elementPaths.every((path) =>
+    MetadataUtils.isFlexLayoutedContainer(MetadataUtils.findElementByElementPath(metadata, path)),
```
Comment (balazsbajorics) [inline editor/src/components/inspector/inspector-common.tsx:139]: this is actually wrong! there's `MetadataUtils.isParentFlexLayoutedContainerForElement()` that should be used if you are talking about whether the element at path is _in_ a flex parent

but if you want to detect if it's in a flex parent -and- participates in the layout, we have the incredibly named `MetadataUtils.isParentYogaLayoutedContainerForElementAndElementParticipatesInLayout`!

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/inspector/inspector-common.tsx (hunk 1 above)
Comment (bkrmendy) [reply editor/src/components/inspector/inspector-common.tsx]: done! (https://github.com/concrete-utopia/utopia/pull/2982/commits/39a3f3293b12d49d8beedcc159aaf27c091a025b)

File: editor/src/components/inspector/inspector-common.tsx (hunk 2 above)
Comment (bkrmendy) [reply editor/src/components/inspector/inspector-common.tsx:158]: (discussed this and it turned out that the name of the function was misleading, I renamed it to `detectAreElementsFlexContainers`)

**Pull request merged**